DOC: suggestions for `scipy.sparse` release notes (#74)
tylerjereddy/scipy: doc/source/release/1.11.0-notes.rst

`scipy.sparse` improvements
===========================
- A new public base class `scipy.sparse.sparray` was introduced, allowing further

>   extension of the sparse array API (like support for 1- and n-dimensional sparse
>   arrays) without breaking backwards compatibility.

Author: ```suggestion
  extension of the sparse array API (such as the support for 1- and
  n-dimensional sparse arrays) without breaking backwards compatibility.
```

>   extension of the sparse array API (such as the support for 1- and
>   n-dimensional sparse arrays) without breaking backwards compatibility.

Author: ```suggestion
  extension of the sparse array API (such as the support for 1-dimensional
  sparse arrays) without breaking backwards compatibility.
```

  `isinstance(x, scipy.sparse.sparray)` to select the new sparse array classes,
  while `isinstance(x, scipy.sparse.spmatrix)` selects only the old sparse
  matrix classes.

> - Division of sparse arrays by a dense array now return sparse arrays

Author: ```suggestion
- Division of sparse arrays by a dense array now return sparse arrays.
```

> - Division of sparse arrays by a dense array now return sparse arrays.

Reviewer: ```suggestion
- Division of sparse arrays by a dense array now returns sparse arrays.
```

> - `scipy.sparse.isspmatrix` now only return `True` for the sparse matrices instances.

Reviewer: ```suggestion
- `scipy.sparse.isspmatrix` now only returns `True` for the sparse matrices instances.
```

  `scipy.sparse.issparse` now has to be used instead to check for instances of sparse
  arrays or instances of sparse matrices.
- Sparse arrays constructed with int64 indices will no longer automatically
  downcast to int32.
- A new `scipy.sparse.diags_array` function was added, which behaves like the
  existing `scipy.sparse.diags` function except that it returns a sparse
  array instead of a sparse matrix.

> - The `argmin` and `argmax` methods now return the correct result when explicit

Reviewer: ```suggestion
- The ``argmin`` and ``argmax`` methods now return the correct result when explicit
```

The double backticks are used in our release notes for any "objects" that aren't fully resolved in the SciPy namespace, so I'll restore those.
